book/classes.md

```diff
@@ -175,13 +420,29 @@ use explicit modules for illustration, but the process is similar when we
 want to define a `.mli` file). In keeping with the usual style for modules,
 we define a type `'a t` to represent the type of our stacks:
 
-<link rel="import" href="code/classes/class_types_stack.ml" />
+```ocaml
+module Stack = struct
+  class ['a] stack init = object
+    ...    
+  end
+
+  type 'a t = 'a stack
+
+  let make init = new stack init
+end
+```
 
 We have multiple choices in defining the module type, depending on how much
 of the implementation we want to expose. At one extreme, a maximally abstract
 signature would completely hide the class definitions:
 
-<link rel="import" href="code/classes/class_types_stack.ml" part="1" />
+```ocaml file=examples/code/classes/class_types_stack.ml,part=1
```

Comment:
Here an example of the new promotion feature of `mdx`: `mdx test` will *not* try to run this code block anymore, it will just check that the code block is in sync with the `ml` file (and it will generate a `.corrected` file if that's not the case).